lib/onc_certification_g10_test_kit/bulk_data_group_export_validation.rb

```diff
@@ -456,7 +456,7 @@ def resource_type
     test do
       title 'Medication resources returned conform to the US Core Medication Profile'
       description <<~DESCRIPTION
-        This test verifies that the resources returned from bulk data export conform to the US Core profiles. This includes checking for missing data elements and value set verification.
+        This test verifies that the resources returned from bulk data export conform to the US Core profiles. This includes checking for missing data elements and value set verification. This test is omitted if bulk data export does not return any Medication resources.#{' '}
```

Comment:
Whats up with that `#{' '}`?  I've seen it in a few places of these descriptions.